frontend/src/api/units.ts

```diff
@@ -106,3 +106,30 @@ export async function createUnit(unit: CreateUnitRequest): Promise<APIResult<Uni
     return handleAPIError(error);
   }
 }
+
+export type RenterCandidate = {
+  firstName: string;
+  lastName: string;
+  phone: string;
+  email: string;
+};
```

Comment:
`RenterCandidate` should have a couple more fields. See [schema](https://docs.google.com/document/d/1ef1bwMTZANkUs5uyz-D93v85jv7CfmoDGwE5ANlX_BE/edit#bookmark=id.77oihdu4yyj0)